bin/check_tron_jobs.py

```diff
@@ -153,24 +153,29 @@ def pretty_print_actions(action_run):
     return display.format_action_run_details(action_run)
 
 
-def get_relevant_run_and_state(job_runs):
-    if len(job_runs['runs']) == 0:
+def get_relevant_run_and_state(job_content):
+    job_runs = sorted(
+        job_content.get('runs', []),
+        key=lambda k: k['run_time'],
```

Comment:
What exactly is run_time, again? Is it possible for it to cause a key error or be None?

Reply:
run_time is the time that job is scheduled to run. It's assigned when the job is scheduled. So it's always there and won't be None.